transport/rif_comms/node.py

```diff
@@ -108,10 +109,13 @@ def _notification_to_message(notification_data: ChannelNewData) -> RaidenMessage
             # we first transform the content of the notification data to a dictionary
             content = json.loads(content_text.decode())
             # the message is inside the notification data, encoded by the RIF Comms GRPC api
-            message_string = bytes(content["data"]).decode()
-            print("Message received", message_string)
-            message_dict = json.loads(message_string)
-            return message_from_dict(message_dict)
+            comms_message_string = bytes(content["data"]).decode()
+            # a comms message can contain one or N raiden messages, therefore it must be parsed
+            messages = validate_and_parse_messages(comms_message_string, None)
+            if not messages:
+                return None
```

Comment:
why are we returning `None` when we expect a list?

my proposal:
```python
    # line 111
    comms_message_string = bytes(content["data"]).decode()
    # a comms message can contain one or N raiden messages, therefore it must be parsed
    return validate_and_parse_messages(comms_message_string, None)
return []
```

Reply:
agree